src/Simulate.js

```diff
@@ -0,0 +1,16 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Random = MissionUtils.Random;
+
+export function processInputToOutput(driverArray, totalPhase) {
+  let result = [];
+  let resultElementObject = {};
+  driverArray.forEach((value) => (resultElementObject[value] = 0));

```

Comment:
```suggestion
  const resultElementObject = driverArray.reduce((acc, cur) => {
    acc[cur] = 0;
    return acc;
  }, {});
```
`reduce` 메서드를 통해 조금 더 선언적으로 작성해보는건 어떨까요??